internal/rdma/rdma-test.go

```diff
@@ -224,3 +228,95 @@ func ValidateRDMAResults(results map[string]string) (bool, error) {
 	// If everything is valid
 	return true, nil
 }
+
+// DeleteMofedDeiverNode delete mofed inventory.
```

Comment:
Typo, rename function name to DeleteMofedDriverNode, or may be something more spcific, like `DeleteMofedRpmDir`